Added YAM GELLO gripper body and fixed link 1 (#21)
wuphilipp/gello_mechanical: README.md

Instructions for assembly can be found here: [Assembly Instructions](https://docs.google.com/document/d/1pzV8LDIGZh6zq8z-ZyKjUZ1ISkdCQctfu_05-ZY95eg/edit?usp=sharing)

## Notes

> If you would like access to the CAD files, please contact me at the email address on my Github profile.

Reviewer: ```suggestion
If you would like access to the CAD files, please contact me at `phil80301[at]berkeley[dot]edu` .
```
